project/plugins.sbt

```diff
@@ -3,3 +3,4 @@
 addSbtPlugin("com.eed3si9n"      % "sbt-buildinfo"    % "0.4.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-scalariform"  % "1.3.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-git"          % "0.8.4")
+addSbtPlugin("me.lessis"         % "bintray-sbt"      % "0.3.0")
```

Comment:
Ok, I'll look into this. Could be nice to have Travis publish snapshot/releases based on tags as part of the build. I'll create a ticket.